lib/vanagon/platform/solaris_10.rb

```diff
@@ -42,14 +42,16 @@ def generate_package(project)
           # - The bin directory and all bin files are owned by root:bin instead of root:sys
           # - All files under lib are owned by root:bin instead of root:sys
           # - All .so files are owned by root:bin instead of root:sys
-          %((cd $(tempdir)/#{name_and_version}; pkgproto . | sort | awk ' \
```

Comment:
@McdonaldSeanp your commit is the one that removes the extra set of parentheses; what is the effect (if any) of removing that second set of parens?